ui/app/components/clients/page/counts.hbs

```diff
@@ -0,0 +1,137 @@
+    {{this.versionText.label}}
+  </h2>
+
+  <div data-test-start-date-editor class="is-flex-align-baseline">
+    {{#if this.formattedStartDate}}
+      <p class="is-size-6" data-test-date-display>{{this.formattedStartDate}}</p>
+      <Hds::Button
+        class="has-left-margin-xs"
+        @text="Edit"
+        @color="tertiary"
+        @icon="edit"
+        @iconPosition="trailing"
+        {{on "click" (fn (mut this.showBillingStartModal) true)}}
+      />
+    {{else}}
+      <DateDropdown @handleSubmit={{this.onDateChange}} @dateType="startDate" @submitText="Save" />
+    {{/if}}
+  </div>
+
+  <p class="is-8 has-text-grey has-bottom-margin-l">
+    {{this.versionText.description}}
+  </p>
+
+  {{#if (eq @activity.id "no-data")}}
+    <Clients::NoData @config={{@config}} @dateRangeMessage={{this.dateRangeMessage}} />
+  {{else if @activityError}}
+    <Clients::Error @error={{@activityError}} />
+  {{else}}
+    {{#if (eq @config.enabled "Off")}}
+      <Hds::Alert @type="inline" @color="warning" class="has-bottom-margin-s" as |A|>
+        <A.Title>Tracking is disabled</A.Title>
+        <A.Description>
+          Tracking is currently disabled and data is not being collected. Historical data can be searched, but you will need
+          to
+          <Hds::Link::Inline @route="vault.cluster.clients.edit">edit the configuration</Hds::Link::Inline>
+          to enable tracking again.
+        </A.Description>
+      </Hds::Alert>
+    {{/if}}
+
+    <div class="is-subtitle-gray has-bottom-margin-m">
+      FILTERS
+      <Toolbar data-test-clients-filter-bar>
+        <ToolbarFilters>
+          <CalendarWidget
+            @startTimestamp={{this.startDate}}
+            @endTimestamp={{this.endDate}}
+            @selectMonth={{this.onDateChange}}
+          />
+          {{#if (or @namespace this.namespaces)}}
+            <SearchSelect
+              @id="namespace-search-select"
+              @options={{this.namespaces}}
+              @inputValue={{if @namespace (array @namespace)}}
+              @selectLimit="1"
+              @disallowNewItems={{true}}
+              @fallbackComponent="input-search"
+              @onChange={{fn this.setFilterValue "ns"}}
+              @placeholder="Filter by namespace"
+              @displayInherit={{true}}
+              class="is-marginless"
+            />
+          {{/if}}
+          {{#if (or @authMount this.authMounts)}}
+            <SearchSelect
+              @id="auth-method-search-select"
+              @options={{this.authMounts}}
+              @inputValue={{if @authMount (array @authMount)}}
+              @selectLimit="1"
+              @disallowNewItems={{true}}
+              @fallbackComponent="input-search"
+              @onChange={{fn this.setFilterValue "authMount"}}
+              @placeholder="Filter by auth method"
+              @displayInherit={{true}}
+            />
+          {{/if}}
+        </ToolbarFilters>
+      </Toolbar>
+    </div>
```

Comment:
One notable difference is that this block used to be behind a conditional where if the empty states were showing then the filters would be hidden. It was decided to remove that so the user could make changes to the filters to trigger a new query and also be able to see what was input that returned no results.